src/frontend/web_application/config/server.default.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  version: process.env.CALIOPEN_VERSION || 'master',
```

Comment:
Version is not be a configuration parameter, but a variable defined during build.

Actually the package.json is updated thru devtools/make_package command

Reply:
right, but I keep it configurable (for example for a specific branch build). So default will read from package.json